src/screens/Main/EditStack/CreatingVideoScreen/styles.tsx

```diff
@@ -36,6 +36,35 @@ const styles = StyleSheet.create({
     width: 307,
     marginBottom: '45%',
   },
+  completedText: {
+    padding: 20,
```

Comment:
should technically be margin since padding is creating extra space within the component unlike margin which creates extra space between the components (which is what we want) It visually has the same effect but best practice margin makes more sense